src/lib.rs

```diff
@@ -124,6 +124,12 @@ impl FromStr for Browser {
     }
 }
 
+pub struct BrowserOptions {
```

Comment:
Please remove `browser` and `url` from this as they should be passed as separate parameters. In a way, `BrowserOptions` should be looked at as a behavioural modifier. Let this contain only `suppress_output` for now. I can see a need for `noblock` at a later time.